Method,Name,Error,Occurrences
GET,/Categorias,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",138
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 7'),429
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 6'),578
GET,/Productos,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",66
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 3'),437
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 8'),654
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 2'),419
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 3'),443
GET,/Categorias,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",321
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 4'),445
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 7'),396
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 1'),423
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 4'),456
GET,/Pedidos/5925,HTTPError('500 Server Error: Internal Server Error for url: /Pedidos/5925'),4
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 12'),483
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 11'),433
GET,/Mesas,RemoteDisconnected('Remote end closed connection without response'),647
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 9'),428
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 3'),418
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 7'),420
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 9'),674
GET,/Productos,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",45
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 9'),452
GET,/Pedidos/5925,RemoteDisconnected('Remote end closed connection without response'),1998
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 12'),461
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 2'),624
GET,/Mesas,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",82
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 11'),443
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 3'),404
GET,/Pedidos/5925,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",108
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 9'),445
GET,/Categorias,HTTPError('500 Server Error: Internal Server Error for url: /Categorias'),6
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 7'),604
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 1'),409
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 10'),588
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 13'),455
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 8'),461
GET,/Pedidos/5925,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",256
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 1'),392
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 12'),403
GET,/Categorias,RemoteDisconnected('Remote end closed connection without response'),2730
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 2'),436
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 13'),465
GET,/Mesas,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",32
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 5'),431
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 8'),455
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 5'),428
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 5'),424
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 1'),385
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 12'),476
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 9'),437
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 10'),407
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 1'),526
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 2'),373
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 8'),430
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 4'),590
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 11'),453
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 2'),407
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 7'),395
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 3'),583
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 5'),622
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 13'),430
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 6'),457
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 9'),424
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 4'),437
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 11'),467
GET,/Productos,RemoteDisconnected('Remote end closed connection without response'),5348
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 11'),435
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 13'),484
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 4'),473
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 2'),458
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 5'),421
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 8'),423
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 1'),385
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 5'),420
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 8'),416
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 4'),455
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 11'),647
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 3'),487
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 7'),460
